queries/Query1.sql

```diff
@@ -0,0 +1,29 @@
+-- List of Borrowed Books: Retrieve all books borrowed by a specific borrower, 
+-- including those currently unreturned.
+
+USE [LibraryManagement]
+GO
+
+-- Specific borrower ID 
+DECLARE @BorrowerID NUMERIC(18, 0) = 5;
+
+SELECT 
+    b.[BookID],
+    b.[Title],
+    b.[Author],
+    b.[ISBN],
+    l.[DateBorrowed],
+    l.[DueDate],
+    l.[DateReturned],
+    CASE 
+        WHEN l.[DateReturned] IS NULL THEN 'Currently Borrowed'
+        ELSE 'Returned'
+    END AS [LoanStatus]
+FROM 
+    [dbo].[Loans] l
+JOIN 
+    [dbo].[Books] b ON l.[BookID] = b.[BookID]
+JOIN 
```

Comment:
I believe we're good to remove the unnecessary second join 
Since Loans already contains the BorrowerID, there's no need to join Borrowers

Reply:
It becomes sth like this:

`DECLARE @BorrowerID INT = 5;

SELECT 
    b.[BookID],
    b.[Title],
    b.[Author],
    b.[ISBN],
    l.[DateBorrowed],
    l.[DueDate],
    l.[DateReturned],
    CASE 
        WHEN l.[DateReturned] IS NULL THEN 'Currently Borrowed'
        ELSE 'Returned'
    END AS [LoanStatus]
FROM 
    [dbo].[Loans] l
JOIN 
    [dbo].[Books] b ON l.[BookID] = b.[BookID]
WHERE 
    l.[BorrowerID] = @BorrowerID;`